CHANGELOG.md

```diff
@@ -19,12 +21,16 @@
 - (app) [\#63](https://github.com/Lorenzo-Protocol/lorenzo/pull/63) Fixed Tendermint client type registration.
 - (client) [\#47](https://github.com/Lorenzo-Protocol/lorenzo/pull/47) Fixed error handling for BTC staking query commands.
 - (x/btcstaking) [\#37](https://github.com/Lorenzo-Protocol/lorenzo/pull/37) Added BTC staking genesis validation.
+- (x/btcstaking) [\#95](https://github.com/Lorenzo-Protocol/lorenzo/pull/95) Remove out-dated check
+- (test) [\#89](https://github.com/Lorenzo-Protocol/lorenzo/pull/89) Fixed testnet cmd
+
```

Comment:
add one fix 

`(x/token) [\#94](https://github.com/Lorenzo-Protocol/lorenzo/pull/94) Allow IBC transfer received if module or pair is disabled.`